lib/rabbit_feed/client.rb

```diff
@@ -160,6 +161,10 @@ def parse_options argv
           opts[:environment] = arg
         end
 
+        o.on '-x', '--prefix VAL', 'Prefix to append to the routing and queue naming allowing exclusive publishing/consuming' do |arg|
```

Comment:
Thinking about it further, I don't like the idea of exposing this as a command line argument as this is config-related and not related to the actual running of the client application.